hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -138,6 +144,18 @@ public HddsDispatcher(ConfigurationSource config, ContainerSet contSet,
             LOG,
             HddsUtils::processForDebug,
             HddsUtils::processForDebug);
+    this.cachedVolumeUsage = CacheBuilder.newBuilder()
+        .maximumSize(1000)
+        .refreshAfterWrite(1, TimeUnit.MINUTES)
```

Comment:
1 minute is probably too aggressive?

Reply:
I agree with @kerneltime here.

Would reducing this to 1 second see much performance impact?

Reply:
1 second should be fine. Need an expiration to prevent it from aggressively refreshing volume stats unnecessarily.

Reply:
made it update 1 second periodically until 1 minute after the last access.